package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "next build",
     "dev": "next dev -p 3040",
-    "gen": "graphql-codegen --config codegen.yml",
+    "gen": "graphql-codegen-esm --config codegen.yml",
```

Comment:
Wihout `-esm`, the code generator throws this error, beuause it tries to use CommonJS
```
Failed to load schema from src/generated/github-schema-loader.js:
      Unable to load from file "/Users/lukas/Documents/UpLeveled/portfolios/next-portfolio-dev/src/generated/github-schema-loader.js": Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/lukas/Documents/UpLeveled/portfolios/next-portfolio-dev/src/generated/github-schema-loader.js from /Users/lukas/Documents/UpLeveled/portfolios/next-portfolio-dev/node_modules/.…
      Instead change the require of github-schema-loader.js in /Users/lukas/Documents/UpLeveled/portfolios/next-portfolio-dev/node_modules/.pnpm/@graphql-tools+code-file-loader@8.0.2_graphql@16.7.1/node_modules/@graphql-tools/code-file-loader/cjs/load-from-module.js to a dynamic import() which is available in all CommonJS modules.
      at /Users/lukas/Documents/UpLeveled/portfolios/next-portfolio-dev/node_modules/.pnpm/@graphql-tools+code-file-loader@8.0.2_graphql@16.7.1/node_modules/@graphql-tools/code-file-loader/cjs/load-from-module.js:34:81
```

In the [Graphql codegen documentation](https://the-guild.dev/graphql/codegen/docs/getting-started/esm-typescript-usage#package-adjustments), there is the fix to use the ESM CLI